MIGRATION.md

```diff
@@ -0,0 +1,324 @@
+# Assumptions
```

Comment:
Please rename the file to something more specific to SCK migration, maybe `docs/migration-from-sck.md`? MIGRATION.md in the root is pretty ambiguous for the combined helm chart where we have O11y+Enterprise.

I would also recommend to add a top level header `# Migration from Splunk Connect for Kubernetes` and move all existing headers to the next level `##`

Reply:
Updating and moving existing headers to the next level